kore/src/Kore/OptionsParser.hs

```diff
@@ -0,0 +1,112 @@
+{- |
+Copyright   : (c) Runtime Verification, 2020
+License     : NCSA
+
+-}
+
+module Kore.OptionsParser
```

Comment:
Please rename this module `Kore.Options` and make it re-export `Options.Applicative`.